fastMatcher.go

```diff
@@ -816,6 +832,15 @@ func (m *FastMatcher) Match(data []byte) (bool, error) {
 	return m.buckets.IsTrue(0), nil
 }
 
+func (m *FastMatcher) MatchWithStatus(data []byte) (bool, int, error) {
```

Comment:
MatchWithStatus() method is used only in the test code (filterExprParser_test.go), is it a place holder and will be used in the future?

Reply:
Yes. XDCR will use this later.